src/pages/EditMonitor.vue

```diff
@@ -764,6 +764,26 @@
                                     </div>
                                 </div>
 
+                                <div class="my-3">
+                                    <div class="form-check">
+                                        <input id="saveResponse" v-model="monitor.saveResponse" class="form-check-input" type="checkbox">
+                                        <label class="form-check-label" for="saveResponse">
+                                            {{ $t("saveResponseForNotifications") }}
+                                        </label>
+                                    </div>
+                                    <div class="form-text">
+                                        {{ $t("saveResponseDescription") }}
+                                    </div>
+                                </div>
+
+                                <div v-if="monitor.saveResponse" class="my-3">
+                                    <label for="responseMaxLength" class="form-label">{{ $t("responseMaxLength") }}</label>
+                                    <input id="responseMaxLength" v-model="monitor.responseMaxLength" type="number" class="form-control" required min="0" step="1">
+                                    <div class="form-text">
+                                        {{ $t("responseMaxLengthDescription") }}
+                                    </div>
+                                </div>
```

Comment:
is this something that benefits from being on the monitor page or should this be just a setting for the entire uptime kuma instance